runtime/lua/vim/lsp/util.lua

```diff
@@ -1082,7 +1093,7 @@ end
 ---  - pad_bottom number of lines to pad contents at bottom
 ---  - separator insert separator after code block
 ---@returns width,height size of float
-function M.fancy_floating_markdown(contents, opts)
+function M.format_fancy_markdown(bufnr, contents, opts)
```

Comment:
Not sold on the name, maybe "stylize_markdown"?

Reply:
`parse_markdown`? `apply_markdown_formatting`?

Reply:
`stylize_markdown` works for me.

Reply:
although, `format_markdown` does also make sense to me. We are literally formatting the markdown to look good in the floating window, by making small textual changes (adding separators, removing the code block delimiters) and applying the syntax highlighting.

Reply:
I dislike "render" because I see a lot on reddit that neovim doesn't "render" our markdown, when in reality it's the application of conceal rules and highlighting, which I don't know if ever have been historically grouped under rendering when describing neovim. Up to you, I think stylize is a superset of formatting which is why I suggested it (because we do format, but we also conceal).

Reply:
ok, sounds good. I'll use stylize